### PR TITLE
Adding nodal data test

### PR DESCRIPTION
**Pull request by pooyan-dadvand**
I am currently in the process of changing the nodal solution step data container, so I'm adding the test for it before starting the changes.

### CODE REVIEW

File: kratos/tests/sources/test_nodal_data.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+			model_part.AddNodalSolutionStepVariable(PRESSURE);
+			model_part.AddNodalSolutionStepVariable(VELOCITY);
+			
+			const std::size_t size = 10;
+			for (std::size_t i = 0; i < size; i++)
+				model_part.CreateNewNode(i, i, 0, 0);
+
+			for (int i = 0; i < repeat; i++) {
+				for (auto i_node = model_part.NodesBegin(); i_node != model_part.NodesEnd(); i_node++) {
+					i_node->GetSolutionStepValue(PRESSURE) = static_cast<double>(i);
```
Comment (RiccardoRossi) [inline kratos/tests/sources/test_nodal_data.cpp]: Use FastGet... here

File: kratos/tests/sources/test_nodal_data.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+			model_part.AddNodalSolutionStepVariable(DISPLACEMENT);
+			model_part.AddNodalSolutionStepVariable(REACTION);
+			model_part.AddNodalSolutionStepVariable(NORMAL);
+			std::size_t size = 10;
+			for (std::size_t i = 0; i < size; i++)
+				model_part.CreateNewNode(i, i, 0, 0);
+
+			for (int i = 0; i < repeat; i++) {
+				for (auto i_node = model_part.NodesBegin(); i_node != model_part.NodesEnd(); i_node++) {
+					i_node->GetSolutionStepValue(PRESSURE) = static_cast<double>(i);
```
Comment (RiccardoRossi) [inline kratos/tests/sources/test_nodal_data.cpp]: FastGet....

**Review by RiccardoRossi — APPROVED** [review]

File: kratos/tests/sources/test_nodal_data.cpp (hunk 1 above)
Comment (pooyan-dadvand) [reply kratos/tests/sources/test_nodal_data.cpp]: Changed!

File: kratos/tests/sources/test_nodal_data.cpp (hunk 2 above)
Comment (pooyan-dadvand) [reply kratos/tests/sources/test_nodal_data.cpp]: Also changed!

**Pull request merged**